src/plugins/data/common/es_query/filters/get_filter_params.ts

```diff
@@ -26,9 +26,10 @@ export function getFilterParams(filter: Filter) {
     case FILTERS.PHRASES:
       return (filter as PhrasesFilter).meta.params;
     case FILTERS.RANGE:
+      const { gte, gt, lte, lt } = (filter as RangeFilter).meta.params;
       return {
-        from: (filter as RangeFilter).meta.params.gte,
-        to: (filter as RangeFilter).meta.params.lt,
+        from: gte || gte === 0 ? gte : gt,
+        to: lt || lt === 0 ? lt : lte,
```

Comment:
This can be simplified.

```suggestion
        from: gte ?? gt,
        to: lt ?? lte,
```